tools/webots.py

```diff
@@ -229,8 +233,10 @@ def exec_run(role, num_of_robots=1, sim_address="127.0.0.1"):
     # Add all robot run commands to process_manager
     for i in range(1, num_of_robots + 1):
 
-        robot_color, port_num = (
-            ("red", 10000 + i) if i <= num_of_robots // 2 else ("blue", 10020 + i - num_of_robots // 2)
+        team_id, robot_color, port_num = (
+            (NUBOTS_TEAM_ID, "red", 10000 + i)
+            if i <= num_of_robots // 2
+            else (ALT_TEAM_ID, "blue", 10020 + i - num_of_robots // 2)
```

Comment:
This hurts my brain a little. Could you please split up this logic so it's easier to follow? (I know you're just working with the same thing which was there, but I think what was there was probably a bit over the top anyway :P )

Reply:
:cry:

Reply:
:brain:  <<  large :weary: